Formula/metview.rb

```diff
@@ -0,0 +1,87 @@
+class Metview < Formula
+  desc "Meteorological workstation software"
+  homepage "https://metview.readthedocs.io/en/latest/"
+  url "https://confluence.ecmwf.int/download/attachments/51731119/MetviewBundle-2022.8.1-Source.tar.gz"
+  version "5.17.1"
+  sha256 "f9eb91918e2bc43388d53cb424de95fa352a3cb64f01cbd0dbd62303a05086f9"
+  license "Apache-2.0"
+
+  # NOTE: do not build on linux because of libtirpc problems - come back to it later
+
+  depends_on "cmake" => :build
+  depends_on "cairo"
+  depends_on "eccodes"
+  depends_on "eigen"
+  depends_on "fftw"
+  depends_on "gdbm"
+  depends_on :macos
+  depends_on "netcdf"
+  depends_on "pango"
+  depends_on "proj"
+  depends_on "qt"
+
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex"  => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..",
+                      "-DBUNDLE_SKIP_ECCODES=1",
+                      "-DECBUILD_LOG_LEVEL=DEBUG",
+                      "-DENABLE_MIR_DOWNLOAD_MASKS=OFF",
+                      "-DENABLE_BUILD_TOOLS=OFF",
+                      "-DENABLE_ECKIT_CMD=OFF",
+                      *std_cmake_args
+      system "make", "install"
+    end
+
+    # Avoid references to Homebrew shims directory
+    # (ecbuild stores some references to the build directory - not used, so we can replace them)
```

Comment:
if they're not used, can they be removed?

Reply:
Good point - we can probably remove most of them - is there a command I can run in the recipe to remove those files?

Reply:
`rm "file"` or `rm_r "dir/"`

Reply:
Thanks, I've changed the formula to remove those files now - much better, as they are not needed!